content/locales/fr.ts

```diff
@@ -73,16 +153,40 @@ export const fr = {
     title: "Identifiants",
     emailBreadcrumb: "Adresses e-mail",
     phoneBreadcrumb: "Numéro de téléphone",
+    emailPlaceholder: "Entrer votre adresses e-mail",
+    phonePlaceholder: "Entrer votre numéro de téléphone",
+    emailInputLabel: "Adresse e-mail *",
+    phoneInputLabel: "Numéro de téléphone *",
+    markIdentity: "Marquer cette identité comme ma principale",
+    addEmail: "Ajouter cette adresse e-mail",
+    addPhone: "Ajouter ce numéro de téléphone",
+    cancel: "Annuler",
   },
   "/account/logins/[type]/[id]": {
     title: "Identifiants",
     emailBreadcrumb: "Adresses e-mail",
     phoneBreadcrumb: "Numéro de téléphone",
+    markEmail: "Marquer cette adresse e-mail comme ma principale",
+    markPhone: "Marquer ce numéro de téléphone comme mon principal",
```

Comment:
```suggestion
    markEmail: "Utiliser en e-mail principal",
    markPhone: "Utiliser en numéro de téléphone principal",
```
Seems more "right" 🤔